tests/framework/AnalyticModels/optimizing/diagonal_valley.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+
+def run(raven, Inputs):
```

Comment:
a brief explaination of this function it would be nice since it is a "analytical model"

Reply:
In addition, aren't we supposed to document the "analytical models" in the analytical latex documentation?

Reply:
I agree, the explanation belongs in the tex and should be added there.